// @shopify/cli/dist/node-package-manager-XAEBCALT.js
import {
  FindUpAndReadPackageJsonNotFoundError,
  PackageJsonNotFoundError,
  UnknownPackageManagerError,
  addNPMDependencies,
  addNPMDependenciesIfNeeded,
  addNPMDependenciesWithoutVersionIfNeeded,
  addResolutionOrOverride,
  bunLockfile,
  checkForCachedNewVersion,
  checkForNewVersion,
  findUpAndReadPackageJson,
  getDependencies,
  getPackageManager,
  getPackageName,
  getPackageVersion,
  inferPackageManager,
  installNPMDependenciesRecursively,
  installNodeModules,
  lockfiles,
  lockfilesByManager,
  npmLockfile,
  packageManager,
  packageManagerFromUserAgent,
  pnpmLockfile,
  pnpmWorkspaceFile,
  readAndParsePackageJson,
  usesWorkspaces,
  versionSatisfies,
  writePackageJSON,
  yarnLockfile
} from "./chunk-4EIIOMWN.js";
import "./chunk-4GAP37VN.js";
import "./chunk-6NVYATES.js";
import "./chunk-CTFDRWUN.js";
import "./chunk-7IK72W75.js";
import "./chunk-3YEPOXGM.js";
import "./chunk-UMUTXITN.js";
import "./chunk-KUJQ4AB6.js";
import "./chunk-G2ZZKGSV.js";
import "./chunk-UV5N2VL7.js";
import "./chunk-XE5EOEBL.js";
import "./chunk-Q2H6D2MH.js";
import "./chunk-EG6MBBEN.js";
import "./chunk-PKR7KJ6P.js";
export {
  FindUpAndReadPackageJsonNotFoundError,
  PackageJsonNotFoundError,
  UnknownPackageManagerError,
  addNPMDependencies,
  addNPMDependenciesIfNeeded,
  addNPMDependenciesWithoutVersionIfNeeded,
  addResolutionOrOverride,
  bunLockfile,
  checkForCachedNewVersion,
  checkForNewVersion,
  findUpAndReadPackageJson,
  getDependencies,
  getPackageManager,
  getPackageName,
  getPackageVersion,
  inferPackageManager,
  installNPMDependenciesRecursively,
  installNodeModules,
  lockfiles,
  lockfilesByManager,
  npmLockfile,
  packageManager,
  packageManagerFromUserAgent,
  pnpmLockfile,
  pnpmWorkspaceFile,
  readAndParsePackageJson,
  usesWorkspaces,
  versionSatisfies,
  writePackageJSON,
  yarnLockfile
};
